attestation-container/README.md

```diff
@@ -0,0 +1,48 @@
+# Attestation Container
+
+This is a gRPC server application to fetch SEV-SNP attestation and collateral.
+
+## Environment
+
+This application needs to run on [SEV-SNP VM](https://www.amd.com/system/files/TechDocs/SEV-SNP-strengthening-vm-isolation-with-integrity-protection-and-more.pdf).
+
+## Dependencies
+
+- [Go](https://go.dev/doc/install)
+- [gRPC](https://grpc.io/docs/languages/go/quickstart/)
+
+## How to start the app
+
+The following command starts the gRPC server application.
+
+```bash
+# In the same directory as this README.md
+go run .
+```
+
+## Test
+
+Unit test:
+
+```bash
+cd attest
+go test
+```
+
+E2E test:
+
+```bash
+# Run the app first
+go run .
+
+# In another terminal
+go test
+```
+
+## Update protobuf
+
+When you edit `.proto` file, you also need to update `.pb.go` files by:
+
+```bash
+protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative protobuf/attestation-container.proto
+```
```

Comment:
This is a comment for myself (and anyone else who was going to suggest this change), rather than a suggestion:

I was going to suggest that this should be auto-generated during the build, rather than built by hand and checked in. But that's deliberately not well supported in Go! It's not easy to make `go build` run arbitrary other commands, and they really _want_ all of the generated source to be checked in, _so that it can be retrieved by dependent modules calling `go get`_!

Reply:
>  It's not easy to make go build run arbitrary other commands

Yeah, looks like making `go` command simple is very prioritized by the author.
https://go.dev/doc/articles/go_command